CryptoTechReminderSystem/UseCase/SendReminder.cs

```diff
@@ -15,11 +16,15 @@ public SendReminder(IMessageSender slackGateway)
 
         public void Execute(SendReminderRequest sendReminderRequest)
         {
-            _slackGateway.Send(new Message
+            var result = _slackGateway.Send(new Message
             {
                 Channel = sendReminderRequest.Channel,
                 Text = sendReminderRequest.Text
             });
+            
+            result.OnSuccess(f => Console.WriteLine($"{sendReminderRequest.Email} was sent a reminder."));
```

Comment:
What if its a ShameLateDevelopers request? It should not log the same message

Reply:
The only messages that are being logged are the people that were sent reminders so I don't think this is an issue.

Reply:
Fixed this in https://github.com/chubberlisk/crypto_tech/pull/37/commits/32ee5927c4d81178afc54603259fe5d688a9116f